utils/mintUtils.ts

```diff
@@ -1,6 +1,6 @@
-import { MassMintNFT } from '../service/scheme'
-import { MediaType } from '../types'
-import { resolveMedia } from '../utils'
+import { MassMintNFT } from '../components/rmrk/service/scheme'
+import { MediaType } from '../components/rmrk/types'
+import { resolveMedia } from '../components/rmrk/utils'
```

Comment:
```suggestion
import { MassMintNFT } from '@/components/rmrk/service/scheme'
import { MediaType } from '@/components/rmrk/types'
import { resolveMedia } from '@/components/rmrk/utils'
```

IMO some of this files should be put outside rmrk folder (maybe /components/base) but it's not on this scope